convert2rhel/subscription.py

```diff
@@ -797,18 +744,14 @@ def disable_repos():
     """Before enabling specific repositories, all repositories should be
     disabled. This can be overriden by the --disablerepo option.
     """
-    disable_cmd = ["subscription-manager", "repos"]
-    disable_repos = []
-    for repo in tool_opts.disablerepo:
-        disable_repos.append("--disable=%s" % repo)
-
-    if not disable_repos:
-        # Default is to disable all repos to make clean environment for
-        # enabling repos later
-        disable_repos.append("--disable=*")
-
-    disable_cmd.extend(disable_repos)
-    output, ret_code = utils.run_subprocess(disable_cmd, print_output=False)
+    cmd = ["subscription-manager", "repos"]
+    disable_cmd = []
+    disabled_repos = ["*"] if not tool_opts.disablerepo else tool_opts.disablerepo
+    for repo in disabled_repos:
```

Comment:
If you want, you can use `get_rhel_disable_repos_command` https://github.com/r0x0d/convert2rhel/blob/main/convert2rhel/repo.py#L65

Reply:
Unfortunately, we can't :( 

`subscription-manager` uses `--disable` instead of `--disablerepo`

Reply:
Ahhh :(